style.css

```diff
@@ -131,8 +131,92 @@ a {
   text-decoration: underline;
 }
 
+/* wrapper for side navbar and component display section  */
+.wrapper {
+  width: 90%;
+  height: 100vh;
+  margin: 0 auto;
+  
+}
+
+.component-container {
+  display: flex;
+}
+
+ /* Side Navbar */
+
+ 
+
```

Comment:
Multiple empty lines could be avoided as that breaks the flow of the code reader/reviewer.